ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/CollectionSelection.tsx

```diff
@@ -235,6 +238,11 @@ function CollectionSelection({
                                 ...oldCollections,
                                 collectionResponse,
                             ]);
+
+                            analyticsTrack({
+                                event: COLLECTION_CREATED,
+                                properties: { source: 'Vuln Reporting' },
```

Comment:
The space caught my eye, and then second look brings question whether to follow left navigation:
**Vulnerability Reporting** versus **Collections** in counterpart event.